pyicloud/services/photos.py

```diff
@@ -213,8 +214,9 @@ def size(self):
 
     @property
     def created(self):
-        dt = datetime.fromtimestamp(self.data.get('createdDate') / 1000.0)
-        return dt.strftime('%Y-%m-%dT%H:%M:%SZ')
+        dt = datetime.utcfromtimestamp(self.data.get('createdDate') / 1000.0)
+        dt = dt.replace(tzinfo=pytz.utc)
```

Comment:
You're right, this can be simplified to

```python
dt = datetime.fromtimestamp(self.data.get('createdDate') / 1000.0, tz=pytz.utc)
```

The time zone needs to specified somewhere or else the `datetime` object will not be time zone aware. I did some experimenting [here](https://gist.github.com/philipbl/dcd4b93f80fbc8f6f8e0dce25b3e49c4).